book/survival.qmd

```diff
@@ -15,100 +15,215 @@ TODO
 
-### Survival Data and Definitions {#sec-surv-set-math}
+## Survival Data and Definitions {#sec-surv-set-math}
 
-Survival analysis has a more complicated data setting than other fields as the 'true' data generating process is not directly modelled but instead engineered variables are defined to capture observed information. Let,
+### Quantifying the Distribution of Event Times {#sec-distributions}
+
+In this section, we introduce different functions that represent a distribution of a random variable and particularly the ones that are particularly important in survival analysis.
+
+For now, assume a continuous, positive, random variable $Y$ t.v.i. $\NNReals$. A standard representation of the distribution of $Y$ is given by the probability density function (pdf), $f_Y: \NNReals \rightarrow \NNReals$, and cumulative distribution function (cdf), $F_Y: \NNReals \rightarrow [0,1]; (\tau) \mapsto P(Y \leq \tau)$.
+
+For reasons that will become apparent later and partly for historical reasons, in survival analysis, it is more common to describe the distribution of event times $Y$ via the so called *survival function* and *hazard function* (often also refered to as *hazard rate*).
+
+The survival function is defined as
+$$
+S_Y: \NNReals \rightarrow [0,1]; \quad
+(\tau) \mapsto P(Y > \tau) = \int^\infty_\tau f_Y(u) \ du,
+$$
+is the probability of not observing an event until some point $\tau \geq 0$ and thus simply the compliment of the cdf: $S_Y(\tau) = 1-F_Y(\tau)$.
+
+The hazard function is given by
+$$
+h_Y: \NNReals \rightarrow \NNReals; \quad
+(\tau) \mapsto  \frac{f_Y(\tau)}{S_Y(\tau)}.
+$$
+
+The hazard function is interpreted as the instantaneous risk of death given that the observation has survived up until that point.
+Note this is not a probability as $h_Y$ can be greater than one.
+
+The cumulative hazard function (chf) can be derived from the hazard function by
+$$
+H_Y: \NNReals \rightarrow \NNReals; \quad
+(\tau) \mapsto \int^\tau_0 h_Y(u) \ du
+$$
+
+The cumulative hazard function relates simply to the survival function by
+$$
+H_Y(\tau) = \int^\tau_0 h_Y(u) \ du = \int^\tau_0 \frac{f_Y(u)}{S_Y(u)} \ du = \int^\tau_0 -\frac{S'_Y(u)}{S_Y(u)} \ du = -\log(S_Y(\tau))
+$$
+
+These last relationships are particularly important, as many methods estimate the hazard rate, which is then used to calculate the cumulative hazard and survival probability
+$$S_Y(\tau) = \exp(-H_Y(\tau)) = \exp\left(-\int_0^\tau h_Y(u)\ du\right).$${#eq-surv-haz}
+
+
+From now on we will often refer to these quantities as $S$, $h$, etc. rather than $S_Y$, $h_Y$ for convenience, unless it's necessary to use the subscript to avoid confusion.
+
+Normally, these quantities could be estimated using standard techniques like regression modeling (estimation of parameters of an assumed distribution).
+However, in contrast to standard settings, $Y$ is only observed partially, due to different types of censoring and truncation describeb below.
```

Comment:
anything in particular you don't like about the previous version?
In the edit, the statement is not true, it can be estimated via MLE, but the likelihood is non-standard.